### PR TITLE
[5266] Make it easy to edit viewed files

### PR DESCRIPTION
**Pull request by aachavarria**
https://github.com/craftercms/craftercms/issues/5266

### CODE REVIEW

File: static-assets/components/cstudio-forms/forms-engine.js [hunk 1]
```diff
@@ -1917,6 +1917,36 @@ var CStudioForms =
             YAHOO.util.Event.addListener(window, 'beforeunload', unloadFn, me);
             YAHOO.util.Event.addListener(closeButtonEl, 'click', cancelFn, me);
 
+            const canEdit = CrafterCMSNext.system.store
+              .getState()
+              .user.permissionsBySite[CrafterCMSNext.system.store.getState().sites.active].includes('content_write');
```
Comment (sumerjabri) [inline static-assets/components/cstudio-forms/forms-engine.js]: Don't we have a set of constants for these strings? What happens when we change these strings in the back?

File: ui/app/src/utils/i18n-legacy.ts [hunk 2]
```diff
@@ -723,6 +723,10 @@ export const formEngineMessages = defineMessages({
   useMVS: {
     id: 'formEngine.useMVS',
     defaultMessage: 'Use _mvs postfix (backward compat)'
+  },
+  contentWritePermission: {
+    id: 'formEngine.contentWritePermission',
+    defaultMessage: "Current user don't have edit permission"
```
Comment (sumerjabri) [inline ui/app/src/utils/i18n-legacy.ts]: The current user doesn't have the necessary permissions to edit this content item"

**Review by sumerjabri — CHANGES_REQUESTED** [review]
What about localization?
Does this cover all the use-cases of editing: FTL, Groovy, text files, etc.?
See other notes.

File: static-assets/components/cstudio-forms/forms-engine.js (hunk 1 above)
Comment (aachavarria) [reply static-assets/components/cstudio-forms/forms-engine.js]: I think we don´t have constants for these strings, another option is use the item available actions

Comment (sumerjabri) [reply static-assets/components/cstudio-forms/forms-engine.js]: That's even better, let's use available actions.

**Comment by aachavarria** [comment]
> What about localization? Does this cover all the use-cases of editing: FTL, Groovy, text files, etc.? See other notes.

that cases are covered on the codeEditor, however I'm doing a update to use available actions for both cases

**Review by rart — APPROVED** [review]

**Pull request merged**